doc/cygnus-common/installation_and_administration_guide/cygnus_agent_conf.md

```diff
@@ -0,0 +1,56 @@
+#<a name="top"></a>Cygnus agent configuration
+Content:
+
+* [Introduction](#section1)
+* [`cygnus_instance_<id>.conf`](#section2)
+* [`agent_<id>.conf`](#section3)
+
+##<a name="section1"></a>Introduction
+Any Cygnus agent is configured through two different files:
+
+* A `cygnus_instance_<id>.conf` file addressing all those non Flume parameters, such as the Flume agent name, the specific log file for this instance, the administration port, etc. This configuration file is not necessary if Cygnus is run as a standlalone application (see later), but it is mandatory if run as a service (see later).
+* An `agent_<id>.conf` file addressing all those Flume parameters, i.e. how to configure the different sources, channels, sinks, etc. that compose the Flume agent behind the Cygnus instance. always mandatory.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/981/commits/76eec6e2ee9eecc6a9fd83458649811f14bcb89b